app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -122,22 +122,22 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public void displayDescriptionActivityData() {
         createTitleView();
-        createParticipantNumberView();
         createDescriptionView();
         createDateView();
         createAddressView();
         createSportView();
         createDurationView();
         createOrganizerView();
         getOrganizerName();
+        createParticipantNumberView();
```

Comment:
Well done, this order is more logical. It doesn't change a lot, but it is better this way, thank you!